### DISCUSSION TITLE
Including incomplete words and sounds.

### DISCUSSION THREAD

**Discussion by Bgerritse** [openai/whisper#855] [General]
Good day! forgive me if already asked but I could not find an answer.
Is there a setting to include stammers/unfinished words/'uuuh's etc in the transcribe result?

**Comment by glangford** [comment]
See this discussion
* https://github.com/openai/whisper/discussions/286

> "The heuristics do not suppress disfluencies by default. It is likely the result of training on data that, presumably, has little to no disfluencies in its transcripts. Disfluencies are in the vocab, so you can try to give it a prompt with the disfluencies you want it to predict.

and if you want to try to predict incomplete words and sounds
* https://github.com/openai/whisper/discussions/288

**Reply by sagar-kris** [reply]
to add to this, maybe play around with the seed parameter and see if one works better for transcribing disfluencies. i've found it has a big impact on the output.

**Reply by LaurinmyReha** [reply]
This variant of Whisper was specifically finetuned to detect fillers and transcribe speech in a verbatim fashion:

https://github.com/nyrahealth/CrisperWhisper

**Comment by LaurinmyReha** [comment]
This variant of Whisper was specifically finetuned to detect fillers and transcribe speech in a verbatim fashion:

https://github.com/nyrahealth/CrisperWhisper

**Comment by Achrafml23** [comment]
You can now detect them using the parameter initial_prompt for example 
initial_prompt= "Umm, Uh, Ah let me think like, hmm... Okay, here's what I'm, like, thinking.".

You can see it in the examples here https://platform.openai.com/docs/guides/speech-to-text/prompting
